docs/jwst/resample/arguments.rst

```diff
@@ -12,30 +12,36 @@ image.
 
 ``--kernel`` (str, default='square')
   The form of the kernel function used to distribute flux onto the output
-  image.
+  image.  Available kernels are `square`, `gaussian`, `point`, `tophat`, `turbo`,
+  `lanczos2`, and `lanczos3`.
+
+``--pixel_scale_ratio`` (float, default=1.0)
+  Ratio of input to output pixel scale.  A value of 0.5 means the output image
+  would have 4 pixels sampling each input pixel.
```

Comment:
I think something here is contradictory.

> Ratio of input to output pixel scale.

So, if input image has pixels with pixel scale of 0.1'' and output image has a finer grid in which pixels have 0.05'', pixel ratio should be `input/output = 2` but second sentence suggests the opposite: `0.5`. This is not right.